Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -1114,33 +1118,268 @@ def export_vulnerabilities_command(args: Dict[str, Any]) -> PollResult:
         return request_uuid_export_vulnerabilities(args)
 
 
+def get_json(response: requests.models.Response) -> dict:
+    '''Handles error status codes and returns the json from response'''
+
+    try:
+        response.raise_for_status()
+        return response.json()
+    except HTTPError as e:
+        demisto.debug(f'Response: {response.text}\nError: {e}')
+        code_messages = {
+            400:
+                'Possible reasons:\n'
+                ' - Your request message is missing a required parameter\n.'
+                ' - The "format" command argument specified an unsupported'
+                ' export format for scan results that are older than 60 days.\n'
+                '   Only Nessus and CSV formats are supported for scan results that are older than 60 days.'
+                ' - Your command arguments included filter query parameters for scan results that are older than 60 days.',
+            404: 'Tenable Vulnerability Management cannot find the specified scan.',
+            429: 'Too Many Requests',
+        }
+        code = response.status_code
+        raise DemistoException(
+            f'Error processing request. Got response status code: {code}' + (
+                f' - {code_messages[code]}'
+                if code in code_messages
+                else f'. Full Response:\n{response.text}')) from e
+    except requests.exceptions.JSONDecodeError as e:
+        demisto.debug(str(e))
+        raise DemistoException(
+            'Error processing request. '
+            'Unexpected response from Tenable IO:'
+            f'\n{response.text}') from e
+
+
+def list_scan_filters_request() -> dict:
+    response = requests.get(
+        f'{BASE_URL}filters/scans/reports',
+        headers=HEADERS, verify=USE_SSL)
+    return get_json(response)
+
+
+def scan_filters_human_readable(filters: list) -> str:
+    context_to_hr = {
+        'name': 'Filter name',
+        'readable_name': 'Filter Readable name',
+        'type': 'Filter Control type',
+        'regex': 'Filter regex',
+        'readable_regex': 'Readable regex',
+        'operators': 'Filter operators',
+        'group_name': 'Filter group name',
+    }
+    return tableToMarkdown(
+        'Tenable IO Scan Filters',
+        [d | d.get('control', {}) for d in filters],
+        headers=list(context_to_hr),
+        headerTransform=context_to_hr.get)
+
+
+def list_scan_filters_command() -> CommandResults:
+
+    response_dict = list_scan_filters_request()
+    filters = response_dict.get('filters', [])
+
+    return CommandResults(
+        outputs_prefix='TenableIO.ScanFilter',
+        outputs_key_field='name',
+        outputs=filters,
+        readable_output=scan_filters_human_readable(filters),
+        raw_response=response_dict)
+
+
+def get_scan_history_request(scan_id, params) -> dict:
```

Comment:
at least for the new http requests, lets make a client that inherits from baseclient